react/src/pages/EndpointDetailPage.tsx

```diff
@@ -429,13 +429,10 @@ const EndpointDetailPage: React.FC<EndpointDetailPageProps> = () => {
     },
     {
       label: t('session.launcher.ModelStorage'),
-      children: (
+      children: endpoint?.model ? (
```

Comment:
How about just using `&&` instead of adding conditional operator(`?`) null when endpoint?.model value is undefined?
And I think is better to change the title of PR since "optional chaining" is used for different situation.

Reply:
For the `&&` operator, any value except null, undefined, and false will be displayed on the screen, so I used the ternary operator to reduce edge cases without directly determining the value.

[teams](https://teams.microsoft.com/l/message/19:14c484402d874dafb15806d093b95a82@thread.skype/1712112975814?tenantId=13c6a44d-9b52-4b9e-aa34-0513ee7131f2&groupId=74ae2c4d-ec4d-4fdf-b2c2-f5041d1e8631&parentMessageId=1712043564875&teamName=devops&channelName=Frontend&createdTime=1712112975814)
[react-dev docs](https://react.dev/learn/conditional-rendering#logical-and-operator-)